internal/rbd/driver.go

```diff
@@ -124,7 +124,7 @@ func (r *Driver) Run(conf *util.Config) {
 	volJournal = journal.NewCSIVolumeJournal(CSIInstanceID)
 	snapJournal = journal.NewCSISnapshotJournal(CSIInstanceID)
 
-	// Initialize default library driver
+	// Initialize default library drivers
```

Comment:
Did `godot` give a warning here? If not, I think this change can be removed.

Reply:
Hey, that's my mistake. Maybe added that while saving the file. will correct that. Thanks for notifying me.